### PR TITLE
Code Golfing

### PR DESCRIPTION
**Pull request by ashfurrow**
Am I using that term correctly?

- Moves to trailing closure syntax where appropriate. 
- Removes "signal" nomenclature.
- Removes unnecessary closure typing.

### CODE REVIEW

File: Kiosk/App/Models/Artwork.swift [hunk 1]
```diff
@@ -39,9 +39,9 @@ final class Artwork: NSObject, JSONAbleType {
     dynamic var images: [Image]?
 
     lazy var defaultImage: Image? = {
-        let defaultImages = self.images?.filter({ (image) -> Bool in
+        let defaultImages = self.images?.filter { (image) -> Bool in
             image.isDefault
-        })
```
Comment (orta) [inline Kiosk/App/Models/Artwork.swift]: couldn't that go into a one liner `$0.isDefault` ?

File: Kiosk/App/Models/SaleArtworkViewModel.swift [hunk 2]
```diff
@@ -130,13 +130,13 @@ extension SaleArtworkViewModel {
     }
 
     func currentBidOrOpeningBid() -> Observable<String> {
-        let signals = [
+        let s = [
```
Comment (orta) [inline Kiosk/App/Models/SaleArtworkViewModel.swift]: `observables`?

File: Kiosk/Bid Fulfillment/ConfirmYourBidPINViewController.swift [hunk 3]
```diff
@@ -5,14 +5,14 @@ import Action
 
 class ConfirmYourBidPINViewController: UIViewController {
 
-    var pin = Variable("")
+    var _pin = Variable("")
```
Comment (orta) [inline Kiosk/Bid Fulfillment/ConfirmYourBidPINViewController.swift]: `private var...`?

**Comment by orta** [comment]
Correct use of the term. Nothing considered a blocker.

File: Kiosk/App/Models/SaleArtworkViewModel.swift (hunk 2 above)
Comment (ashfurrow) [inline Kiosk/App/Models/SaleArtworkViewModel.swift]: Ha, this must've got caught up in my find/replace "signal" with "" sweep :)

**Pull request merged**